app/assets/javascripts/notifications.js

```diff
@@ -109,6 +114,17 @@ function quickCreate(form, data_type) {
   });
 }
 
+function closeQuickCreate() {
+  if ($(this).closest('.quick_create').attr('id') === 'category_quick_create' && !$('#category_quick_create').hasClass('display_none')) {
```

Comment:
Just went back and refactored this, I also needed to wrap the quickCreateId in $() to use jquery methods on it.